pyproject.toml

```diff
@@ -122,6 +122,9 @@ version-file = { "script" = "_packaging.make-script:make_file_version_info" }
 pyinstall = { "script" = "_packaging.make-script:make_pyinstaller" }
 package = ["install-pyinstall", "gaphor-script", "version-file", "pyinstall"]
 win-installer = { "script" = "_packaging.windows.build-win-installer:main" }
+docker-build = "docker build -t ghcr.io/gaphor/gaphor-appimage _packaging/appimage"
+docker-push = "docker push ghcr.io/gaphor/gaphor-appimage:latest"
+docker-run = "docker run --rm -it --volume $PWD:/gaphor ghcr.io/gaphor/gaphor-appimage"
```

Comment:
Nice solution to use ghcr :+1: This potentially saves a lot of time. A shame we can't use that for our Windows build.

I would call these targets `appimage-something`, since that's what we're building. I would also rename the workflow yaml file to something appimage-ish, as well.

Reply:
@amolenaar Thanks, I updated the names to prefix them with appimage :+1: and rebased the PR, so we can make sure that the new content is working with the AppImage as well.